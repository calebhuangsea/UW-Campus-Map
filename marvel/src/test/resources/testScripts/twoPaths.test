# Find a path with where there are 2 paths with equal distance
# between start node and end node

LoadGraph twoPaths twoPaths.csv

FindPath twoPaths n1 n3