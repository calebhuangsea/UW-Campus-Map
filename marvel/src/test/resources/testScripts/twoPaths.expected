# Find a path with where there are 2 paths with equal distance
# between start node and end node

loaded graph twoPaths

path from n1 to n3:
n1 to n2 via a1
n2 to n3 via a2